app/services/task.py

```diff
@@ -1,10 +1,10 @@
 from app.models import Task
 from app.config.db_config import SessionLocal
 
-def add(title,description):
+def add(data):
```

Comment:
It will be better if you add all the params you needed instead of unpacking them here after starting a db session